File: Solved/Namayeshgah Majazi/NamayeshgahMajazi.java
import java.util.Scanner;

/*
Question title = namayeshgah majazi - نمایشگاه مجازی
https://quera.ir/problemset/110015/
 */

public class NamayeshgahMajazi {
    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);
        int count = Integer.parseInt(sc.nextLine().trim());

        //number of booth
        int num = 1;

        int remainingBooth = count;
        for (int i = 1; i <= 4; i++) {
            System.out.println("########.......########");
            switch (count) {
                case 1: {
                    System.out.println("#ghorfe" + num + "..............#");
                    remainingBooth--;
                    break;
                }
                case 0: {
                    System.out.println("#.....................#");
                    break;
                }
                default: {
                    System.out.println("#ghorfe" + num + ".......ghorfe" + (1 + num) + "#");
                    remainingBooth -= 2;
                    num += 2;
                    break;
                }
            }
            count = remainingBooth;
        }
        System.out.println("#######################");
    }
}

File: Solved/Namayeshgah Majazi/Namayeshgah Majazi.py
# نمایشگاه مجازی
# ID : 110015
# https://quera.org/problemset/110015/
# https://b2n.ir/z77955


count = int(input())
num = 1
remainingBooth = count

for i in range(1, 5):
    print("########.......########")

    if count == 1:
        print(f"#ghorfe{num}..............#")
        remainingBooth -= 1

    elif count == 0:
        print("#.....................#")

    else:
        print(f"#ghorfe{num}.......ghorfe{num + 1}#")
        remainingBooth -= 2
        num += 2

    count = remainingBooth

print("#######################")
